Testscripts/Linux/verify_vmbus_heartbeat_properties.sh

```diff
@@ -55,18 +55,27 @@ SetTestStateRunning
 test_failed=""
 
 # check the file structure exists and that all the files can be read
-vmbus_driver_files=(channel_vp_mapping class_id client_monitor_conn_id \
+vmbus_driver_files_1=(channel_vp_mapping class_id client_monitor_conn_id \
```

Comment:
Could we add some comments to avoid the magic suffix 1 or 2 for variable 'vmbus_driver_files_'?

Reply:
@harz566 is right. Adding a number to a variable is not good practice.